### PR TITLE
Fix comment indicators not hiding when (un)resolving

### PR DESCRIPTION
**Pull request by frcroth**

### CODE REVIEW

**Review by T4rikA — APPROVED** [review]

File: lively.collab/comments/commentMorph.js [hunk 1]
```diff
@@ -184,7 +184,9 @@ export class CommentMorph extends Morph {
   initializeCommentIndicator () {
     this.commentIndicator = new CommentIndicator(this, this.comment, this.referenceMorph);
     this.commentIndicator.fontColor = this.comment.isResolved() ? Color.rgb(174, 214, 241) : Color.rgb(241, 196, 15);
-    if (CommentBrowser.isOpen()) this.showCommentIndicator();
+    if (CommentBrowser.isOpen() && (this.comment.isResolved() == CommentBrowser.instance.resolvedFilter)) {
```
Comment (SilvanVerhoeven) [inline lively.collab/comments/commentMorph.js:197]: What about a method like `showsResolvedComments`?

Comment (linusha) [inline lively.collab/comments/commentMorph.js:197]: I am not a fan of the name `resolvedFilter`. What about `isInResolvedMode` or `resolvedFilterActive`?

Comment (SilvanVerhoeven) [reply lively.collab/comments/commentMorph.js:197]: Will be done after merge

**Comment by frcroth** [comment]
TODO Change name of resolvedFilter

**Pull request merged**